File: Day 04 - 03.04.2019/TypeScript-Factory(Design-Patterns)/Cashier.ts
import {IUser} from "./IUser";

export class Cashier implements IUser
{
	public static isValid( name?:string,  password?:String):boolean
	{
		return name==("cash") &&password==("1234");
				
	}

	
	public getInfo():string
	{
		return "I am a Cashier";
	}
}

File: Day 04 - 03.04.2019/CSharp-Factory(Design-Patterns)/Cashier.cs
using System;

namespace Factory__Design_Patterns_
{
    class Cashier:IUser
    {
        public static bool isValid(String name, String password)
        {
            return name==("cash") &&password==("1234");
                    
        }

        
    public  String getInfo()
        {
            return "I am a Cashier";
        }
    }
}
